Documentation/Readme.md

```diff
@@ -0,0 +1,23 @@
+# Documents Index 
+
+This is the developer documentation for the Windows Communication Foundation repository. 
+
+## Learn about the project:
+
+- How to [build the repository](Developer-Guide.md#building-the-repository).
+- How to [report issues](Developer-Guide.md#reporting-issues).
+- [Contribution guidelines](Contributing.md)
+- [Project priorities](Developer-Guide.md#project-goals-and-priorities)
+- [Developer Guide](Developer-Guide.md)
+- [Contributing](Contributing.md)
+- [Issue Guide](Issue-Guide.md)
+
+### Available WCF Libraries
+  * [Primitives](System.ServiceModel.Primitives.md)
+  * [Http](System.ServiceModel.Http.md)
+  * [NetTcp](System.ServiceModel.NetTcp.md)
+  * [Duplex](System.ServiceModel.Duplex.md)
+  * [Security](System.ServiceModel.Security.md)
+
```

Comment:
I originally abbreviated this names because they didn't fit in the wiki's sidebar.  But now they could be spelled out long-hand, like [System.ServiceModel.Primitives](System.ServiceModel.Primitives.md).  Your call on which seems clearer.